x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/jobs_list.tsx

```diff
@@ -103,6 +176,30 @@ export function JobsList({ data, status, onAddEnvironments }: Props) {
             </h2>
           </EuiTitle>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
+            checked={showLegacyJobs}
+            onChange={(e) => {
+              setShowLegacyJobs(e.target.checked);
+            }}
+            label={i18n.translate(
+              'xpack.apm.settings.anomalyDetection.jobList.showLegacyJobsCheckboxText',
+              {
+                defaultMessage: 'Show legacy jobs',
+              }
+            )}
+          />
```

Comment:
Can we make the switch toggled on by default to show legacy jobs. I found that when I see the update callout and go to the page, I don't see my legacy jobs that needs to be updated.

<img width="1182" alt="CleanShot 2021-12-03 at 14 15 20@2x" src="https://user-images.githubusercontent.com/4104278/144609010-4a3e05c7-d2ae-48cb-904d-4ddb4b0cae94.png">

If we instead toggle it on by default, they'll show up as expected. The user can choose to toggle it off once they're done updating and make the table more manageable.

<img width="1185" alt="CleanShot 2021-12-03 at 14 15 26@2x" src="https://user-images.githubusercontent.com/4104278/144609120-9fec8731-b47b-4405-a65c-90e049f78241.png">

Reply:
It now shows legacy jobs if there are no other jobs. That seems like a reasonable compromise, without having to implement some setting mechanism backed by localStorage.